test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -57,32 +65,25 @@ def _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
     # Feed args and kwargs into exporter.
     # Note that exporter should flatten kwargs into positional args the exported model;
     # since ONNX doesn't represent kwargs.
-    onnx_model = fx_onnx.export_after_normalizing_args_and_kwargs(
-        model,
-        *input_args,
-        opset_version=opset_version,
-        use_binary_format=True,
-        enable_dynamic_axes=True,
-        **input_kwargs,
+    exporter = DynamoOptimizeExporter(
+        options=ExportOptions(opset_version=opset_version, dynamic_shapes=True),
+        model=model,
+        model_args=input_args,
+        model_kwargs=input_kwargs,
     )
 
-    # Inspect the model's signature. It will be used
-    # to flatten kwargs.
-    if isinstance(model, torch.nn.Module):
-        signature = inspect.signature(model.forward)
-    else:
-        signature = inspect.signature(model)
+    export_output = exporter.export()
```

Comment:
Users (and tests) should use public interface (`dynamo_export`) as much as possible to 1) flush out UX issues, 2) limitations and 3) set an example on how to properly do stuff

Reply:
That is indeed a goal.